frontend/interfaces/software.ts

```diff
@@ -306,6 +308,7 @@ export interface ISoftwareInstallVersion {
 
 export interface IHostSoftwarePackage {
   name: string;
+  automatic_install: boolean;
```

Comment:
@lucasmrod Will this return? or is it just for post

Reply:
https://github.com/fleetdm/fleet/pull/24729#discussion_r1882928228